### PR TITLE
Added some more metadata files

### PR DESCRIPTION
**Pull request by duncanmmacleod**
### Summary

This PR adds the following files

- `CONTRIBUTING.md` - adapted from the LALSuite contributions guide
- `MANIFEST.in` - which lists non-standard files that should be included in sdist tarballs
- `.github/CODEOWNDERS` (see below)

### `CODEOWNERS`

See https://help.github.com/articles/about-codeowners/ for what the file does.

At the moment I've just added myself to be auto-assigned to review any changes to build and test related files. Currently there isn't a 'default code owner' for all files, but we might want to set @cdcapano to be auto-assigned to review any python changes, since he wrote most of the code to start with, or possibly @cmbiwer.

### CODE REVIEW

**Comment by vivienr** [comment]
I agree. We should also enable:

- [x] **Require review from Code Owners**
Require an approved review in pull requests including files with a designated code owner.

in the repository's settings, no?

**Comment by duncanmmacleod** [comment]
@vivienr, agreed.

**Comment by vivienr** [comment]
How about for now we leave the default owner off, so that any organisation member can approve? If @cdcapano and/or @cmbiwer would rather be as default, they can be added.

**Review by vivienr — APPROVED** [review]
All good !

**Pull request merged**
